File: ts-sdk/src/sdk/models/shared/walletserrorresponse.ts
import { SpeakeasyBase, SpeakeasyMetadata } from "../../../internal/utils";

export enum WalletsErrorResponseErrorCodeEnum {
    Validation = "VALIDATION"
}

export class WalletsErrorResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=errorCode" })
  errorCode: WalletsErrorResponseErrorCodeEnum;

  @SpeakeasyMetadata({ data: "json, name=errorMessage" })
  errorMessage: string;
}

File: python-sdk/src/formanceapi/models/shared/walletserrorresponse.py
import dataclasses
from dataclasses_json import dataclass_json
from enum import Enum
from formanceapi import utils

class WalletsErrorResponseErrorCodeEnum(str, Enum):
    VALIDATION = "VALIDATION"


@dataclass_json
@dataclasses.dataclass
class WalletsErrorResponse:
    error_code: WalletsErrorResponseErrorCodeEnum = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('errorCode') }})
    error_message: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('errorMessage') }})
    